cloudflare.go

```diff
@@ -122,8 +136,13 @@ func (api *API) request(method, uri string, reqBody io.Reader) (*http.Response,
 
 	// Apply any user-defined headers first.
 	req.Header = cloneHeader(api.headers)
-	req.Header.Set("X-Auth-Key", api.APIKey)
-	req.Header.Set("X-Auth-Email", api.APIEmail)
+	if api.authType&AuthKeyEmail != 0 {
```

Comment:
Yeah, `gofmt` uses the spacing in statements like the above to show the precedence order. Oh well.